ui/src/services/alertUtils.js

```diff
@@ -0,0 +1,208 @@
+          removeIndex.push(i);
+        } else if (alerts[j].labels.severity === STATUS_WARNING) {
+          removeIndex.push(j);
+        }
+      }
+    }
+  }
+
+  let removedWarningAlerts = [...alerts];
+  removeIndex.forEach((index) => removedWarningAlerts.splice(index, 1));
+
+  return removedWarningAlerts;
+};
+
+// Sort the alerts base on the `severity`
+export const sortAlerts = (alerts: AlertList): AlertList => {
+  return alerts.sort(function (a, b) {
+    return compareHealth(a.labels.severity, b.labels.severity);
+  });
+};
+
+export const isObjectHasProperty = (obj: {}, property: string): boolean => {
+  return Object.prototype.hasOwnProperty.call(obj, property);
+};
+
+/* 
+Format the alerts
+If the label has multiple values, reshape it from comma-separated to an array
+Nets-alert:
+  node: 'node1'
+  selectors: ['node', '...']
+  parents: ['Node', '...']
+  comp: 'nets' 
+*/
+export const formatAlerts = (alerts: Array<PrometheusAlert>): AlertList => {
+  return alerts.map((alert) => {
+    let parents = [];
+    let selectors = [];
+
+    if (isObjectHasProperty(alert.labels, 'parents')) {
+      parents.push(alert.labels.parents.split(','));
+    }
+    if (isObjectHasProperty(alert.labels, 'selectors')) {
+      selectors.push(alert.labels.selectors.split(','));
+    }
+    return {
+      summary: alert.annotations.summary,
+      description: alert.annotations.description || alert.annotations.message,
+      startsAt: alert.startsAt,
+      endsAt: alert.endsAt || new Date().toISOString(),
+      severity: alert.labels.severity,
+      documentationUrl: alert.annotations.runbook_url,
+      labels: { ...alert.labels, parents, selectors },
+    };
+  });
+};
+
+/* 
+This function is used when filtering the alerts, by checking if the `labels` is 
+the subset of the `filters` object. 
+Note that: The values of the selectors can be more than one, so it's possible to be an array.
+
+e.g. alertname: [NODE_FILESYSTEM_SPACE_FILLINGUP, NODE_FILESYSTEM_ALMOST_OUTOF_SPACE, 
+NODE_FILESYSTEM_FILES_FILLINGUP, NODE_FILESYSTEM_ALMOST_OUTOF_FILES]  
+*/
+export const isAlertSelected = (
+  labels: {
+    [labelName: string]: string,
+    parents?: string[],
+    selectors?: string[],
+  },
+  filters: {
+    [labelName: string]: string | string[],
+    parents?: string[],
+    selectors?: string[],
+  },
+): boolean => {
+  return Object.getOwnPropertyNames(filters)
+    .map((key) => {
+      if (!filters[key] || !labels[key] || !isObjectHasProperty(labels, key))
+        return false;
+      if (Array.isArray(filters[key])) {
+        if (Array.isArray(labels[key])) {
+          return (
+            filters[key].find((val) => {
+              return labels[key].indexOf(val) >= 0;
+            }) !== undefined
+          );
+        } else return filters[key].includes(labels[key]);
+      } else {
+        return labels[key].includes(filters[key]);
+      }
+    })
+    .reduce((boolA, boolB) => {
+      return boolA && boolB;
+    });
```

Comment:
I would add an explicit initial value here to ease readability
```suggestion
}, true);
```